docs/content/1.getting-started/7.i18n/2.vue.md

```diff
@@ -32,6 +34,24 @@ import { fr } from '@nuxt/ui/locale'
 </template>
 ```
 
+### Direction
+
+Each locale has a default direction, but you can override it using the `dir` prop if needed.
+
+Use the `dir` prop with `ltr` or `rtl` to set the global reading direction of your app:
+
+```vue [App.vue]
+<script setup lang="ts">
+import { fr } from '@nuxt/ui/locale'
```

Comment:
@benjamincanac 
```
import { fr } from '@nuxt/ui/locale'
</script>
<template>
  <UApp dir="ltr" :locale="fr">
    <RouterView />
  </UApp>
</template>
``` 
OR
```
import { ar } from '@nuxt/ui/locale'
</script>
<template>
  <UApp dir="rtl" :locale="ar">
    <RouterView />
  </UApp>
</template>
```

Reply:
What do you mean?

Reply:
The `fr` locale is `ltr` and `ar` locale is `rtl`, the example demonstrates how to override the injected dir by the locale so it should be reversed. This is why I specify `fr` with `rtl`.

Reply:
@benjamincanac 
The`french` language is written from left to right (LTR), not right to left (RTL).